src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayRenta = 40;
+  let totalCost = days * dayRenta;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
+  }
```

Comment:
Prefer 'if' with 'return' over 'if else' to simplify the conditions. Also, avoid using 'else' after 'if' with 'return' as the code after 'if' won't be executed anyway.